src/applications/disability-benefits/all-claims/tests/data

```diff
@@ -0,0 +1 @@
+fixtures/data
```

Comment:
The `data` directory was moved to `fixtures/data` for consistency, but I added a symlink at the original location for backwards compability. It should be removed along with old tests as they get converted.